### ISSUE TITLE
Consider breaking flake.nix into more digestible files

### ISSUE THREAD

**Issue by eureka-cpu** [cpick/nix-rosetta-builder#6]
Just a suggestion, it would greatly improve readability if things were in separate files then imported into the flake. If you're open to it, I wouldn't mind doing this and of course happy to take constructive criticism. I'll be giving this flake a test run here in the near future, really excited to try it. Props for doing this!

**Comment by kradalby** [comment]
I agree this looks awesome, it looks like it would have some pretty good building blocks for running NixOS as a "all purpose" VM as well, for example with a docker mode, so you can replace colima and other "non predictable" parts of the  chain.

**Comment by cpick** [comment]
> it would greatly improve readability if things were in separate files then imported into the flake.

Yes, I agree.

> I wouldn't mind doing this

Please do! For the time being I want to keep configuration options to a minimum so that we can get some feedback from multiple people running it before the configurations multiply, but there’s no reason the preset values couldn’t be passed between files.

> I'll be giving this flake a test run here in the near future, really excited to try it. Props for doing this!

Happy to hear, please let me know how it goes.

**Comment by cpick** [comment]
I split flake.nix into separate pieces.  I'm still learning Nix so any suggestions on making it more idiomatic would be welcome.
